sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Azure.Messaging.EventHubs.Processor.csproj

```diff
@@ -13,7 +13,11 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Messaging.EventHubs" VersionOverride="5.2.0-preview.3" /> <!--Version override exists to bind to the preview; this will be removed when v5.2.0 is released for GA -->
```

Comment:
This reference and the project version will be updated for release in a separate PR, once the `Azure.Messaging.EventHubs` package for September has been published.